packages/accordion/src/styled.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from '@littlespoon/theme/lib/colors'
+import { p3, p4 } from '@littlespoon/theme/lib/fonts/paragraph'
+import { family, weight } from '@littlespoon/theme/lib/fonts/primary'
+import styled from 'styled-components'
+
+export const Wrapper = styled.div<{
+  isExpanded: boolean
+}>`
+  position: relative;
+  height: ${(props) => (props.isExpanded ? `auto` : '0')};
+  transition: all 0.4s ease-in-out;
```

Comment:
Can you confirm if transitions would work in this case? Unless something changed In recent years, there is no way to transition from `0` to `auto`.

 For that to work, you usually need to get the height of the hidden content with JS so you can transition it from `0` to `x`.

Here is a reference of what we use on the FE for this.
https://github.com/little-spoon-dev/little-spoon-frontend/blob/main/app/components/utility/ExpandableContent/index.js